### DISCUSSION TITLE
Are collection-scoped variables available or planned?

### DISCUSSION THREAD

**Discussion by ams10961** [usebruno/bruno#298] [Ideas]
Bruno looks great, thanks for making it available.

One key feature that I haven't found yet is collection-scoped variables - have I missed this, or is this planned?

**Comment by jakoss** [comment]
I would widen the question  to collection wide auth and scripts

**Reply by answerquest** [reply]
It looks like auth and scripts are covered in now, but collection variables aren't there yet.

**Comment by igel1** [comment]
https://docs.usebruno.com/scripting/javascript-reference.html   ->  search for "Collection Variables"
(worked very well for me ...)

In addition: when you klick on the eye-icon (upper right corner), collection variables will be displayed
... and you can read in the lower buttom section:

"Note: As of today, collection variables can only be set via the API - getVar() and setVar().
In the next release, we will add a UI to set and modify collection variables."

**Comment by jakoss** [comment]
Also, there is open feature request for this https://github.com/usebruno/bruno/issues/334

**Reply by answerquest** [reply]
that didn't cover collection variables

**Comment by W0nderMuffin** [comment]
Where are collection variables stored?
Is it possible to write them directly to the collection.bru file like in environment .bru files via `vars {...}`? 


I can't find anything helpful if I want to store collection variables without using the UI or javascript runtime.
